client/src/components/Employees/Employees.js

```diff
@@ -33,7 +33,6 @@ function AddEmployee() {
   };
 
   function submitClick() {
-    console.log("In click function");
```

Comment:
Let's delete this (it's not adding anything useful to the log)

Reply:
done